src/hevm/src/EVM.hs

```diff
@@ -455,14 +455,15 @@ initialContract theContractCode = Contract
       keccak (stripBytecodeMetadata theCode)
   , _storage  = Concrete mempty
   , _balance  = 0
-  , _nonce    = 0
+  , _nonce    = if creation then 1 else 0
```

Comment:
KEVM also sets the nonce to 1 for newly created accounts as part of `#mkCreate` [here](https://github.com/kframework/evm-semantics/blob/master/evm.md#account-creationdeletion).
We are not always setting it to 1, however, because EOAs start with the nonce at 0.

Reply:
makes sense, thanks!